mantle/platform/machine/qemu/cluster.go

```diff
@@ -149,6 +149,12 @@ func (qc *Cluster) NewMachineWithQemuOptions(userdata *conf.UserData, options pl
 		primaryDisk = *diskp
 	}
 
+	if options.Cex {
```

Comment:
For consistency, let's make this

```suggestion
	if qc.flight.opts.Cex || options.Cex {
```

That way e.g. if someone runs `kola run basic --qemu-cex`, it'll run the basic tests with the CEX device attached. Other options work that way as well.